static/app/components/searchQueryBuilder/tokens/filter/parsers/string/grammar.pegjs

```diff
@@ -14,21 +14,55 @@ item = s1:spaces c:comma s2:spaces value:(!comma text_in_value)? {
 }
 
 text_in_value
-  = quoted_value / in_value / empty_value
+  = quoted_contains_value
+  / quoted_value
+  / contains_value
+  / in_value
+  / empty_value
 
 empty_value
   = spaces {
-    return tc.tokenValueText(text(), false);
+    if (tc.enableContainsCheck()) {
+      return tc.tokenValueText('', false, false);
+    }
+    return tc.tokenValueText('', false);
```

Comment:
Did we lose the `text()` ?

Reply:
Went with the later suggestion, avoiding changes to the parser.